lib/cli/src/commands/app/list.rs

```diff
@@ -34,6 +34,19 @@ pub struct CmdAppList {
     /// Asks whether to display the next page or not
     #[clap(long, default_value = "false")]
     paging_mode: bool,
+
+    /// Sort order for apps.
+    ///
+    /// Use: --newest, --oldest or --last-updated
```

Comment:
Doesn't clap print possible values already? Maybe it's different from `ValueEnum`'d enums.

Reply:
It didn't use to, they might have added it.